src/wasm.h

```diff
@@ -1160,12 +1160,17 @@ struct Importable {
 
 class Function;
 
+// Represents an offset into a wasm binary file. This is used for debug info.
```

Comment:
This might say "offset into a wasm binary file or section" since that's what it actually is.